.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 79
-ignore = E203, E231, W503, F723, F821
```

Comment:
Could you explain why you removed E231, F723, F821 from `ignore`?

Reply:
In short, because we don't have any of those errors! Here's a description of those errors
- [`E231`](https://www.flake8rules.com/rules/E231.html) - Missing whitespace after ',', ';', or ':'
- [`F723`](https://flake8.pycqa.org/en/4.0.1/user/error-codes.html) - syntax error in type comment
- [`F821`](https://www.flake8rules.com/rules/F821.html) - Undefined name name